# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../ncompress/config.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# Copyright (C) 1998 - 2003 ROCK Linux Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---


ncompress uses a really nice build script with a few evil bugs ...

 - Clifford

--- ./build	Wed Oct 28 12:10:53 1992
+++ ./build.new	Mon Aug 31 21:25:22 1998
@@ -278,8 +278,6 @@
 		[ -f /usr/include/sys/dir.h ] && { SYSDIR=yes; }
 		[ -f /usr/include/dirent.h ] && { DIRENT=yes; }
 		[ -f /usr/include/utime.h ] && { UTIME_H=yes; }
-		[ -d /usr/local/bin ] && { BINDIR=/usr/local/bin; }
-		[ -d /usr/local/man ] && { BINDIR=/usr/local/man/man1; }
 		[ -f /usr/bin/compress ] && { BINDIR=/usr/bin; }
 	
 		if [ ".${CC}" = . ]
@@ -790,7 +788,7 @@
 
 
 compress:	Makefile compress42.c patchlevel.h
-	$(CC) -o compress $(options) "-DCOMPILE_DATE=\"\`date\`\"" compress42.c $(LBOPT)
+	\$(CC) -o compress \$(options) "-DCOMPILE_DATE=\\"\`date\`\\"" compress42.c \$(LBOPT)
 
 install:	compress
 		[ -f \$(BINDIR)/compress ] && \\
